modules/end-user-portal/src/Models/NoResultsModel.cs

```diff
@@ -0,0 +1,26 @@
+using GpConnect.NationalDataSharingPortal.EndUserPortal.Helpers;
+using GpConnect.NationalDataSharingPortal.EndUserPortal.Helpers.Enumerations;
+using GpConnect.NationalDataSharingPortal.EndUserPortal.Models;
+using GpConnect.NationalDataSharingPortal.EndUserPortal.Resources;
+using Microsoft.AspNetCore.Mvc;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace GpConnect.NationalDataSharingPortal.EndUserPortal.Pages.Search;
+
+public partial class NoResultsModel : BaseModel
+{
+    [BindProperty(SupportsGet = true)]
+    public string Query { get; set; } = "";
+
+    [BindProperty(SupportsGet = true)]
+    public SearchMode Mode { get; set; } = SearchMode.Name;
+
+    public string SearchType => Mode == SearchMode.Name ? SearchMode.Name.GetNameParameter() : SearchMode.Code.GetNameParameter();
+
+    [Display(Name = "OrganisationNameNoResults", ResourceType = typeof(DataFieldNameResources))]
```

Comment:
Can we move this to the SearchMode enum please

Reply:
Sorted.